<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_dataentry_dataentry___batch_condition</class>
 <widget class="QWidget" name="form_dataentry_dataentry___batch_condition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1036</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Общие данные</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Все даты:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>New</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Дата:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Контакт</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_2">
          <item>
           <property name="text">
            <string>admin</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Кондиция партии</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <column>
         <property name="text">
          <string>Номер</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Коготь</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Тело</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Грудь</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Вес</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
